### PR TITLE
Link to Chrome extension on Chrome (React)

### PR DESCRIPTION
**Pull request by Vinnl**
# New feature description

This is the React equivalent of https://github.com/mozilla/fx-private-relay/pull/1615.

# Screenshot (if applicable)

image

image

# How to test

Same as #1615. See https://deploy-preview-1616--fx-relay-demo.netlify.app/?mockId=some and https://deploy-preview-1616--fx-relay-demo.netlify.app/?mockId=onboarding in Chrome.

# Checklist

- [x] l10n dependencies have been merged, if any. - No: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/65
- [x] All acceptance criteria are met.
- [x] I've added or updated relevant docs in the docs/ directory.
- [x] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/static/scss/libs/protocol/css/includes/tokens/dist/index.scss`).
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: frontend/src/components/dashboard/PremiumOnboarding.tsx [hunk 1]
```diff
@@ -384,21 +388,69 @@ const StepThree = () => {
   const { l10n } = useLocalization();
   const isLargeScreen = useMinViewportWidth("md");
 
+  let addonDescription = null;
+  let title = l10n.getString("multi-part-onboarding-reply-headline");
```
Comment (groovecoder) [inline frontend/src/components/dashboard/PremiumOnboarding.tsx:392]: `let` and `if/else if` smells bad to me. So I took a stab at refactoring this into new more-declarative components [here](https://github.com/mozilla/fx-private-relay/pull/1708). (And I learned a lot about React and TypeScript on the way!)

**Review by groovecoder — CHANGES_REQUESTED** [review]
I think I like my changes in https://github.com/mozilla/fx-private-relay/pull/1708 ?

File: frontend/pendingTranslations.ftl [hunk 2]
```diff
@@ -80,6 +80,17 @@ tips-footer-link-feedback-tooltip = Give feedback
...
+## https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/65
+-brand-name-chrome = Chrome
+-brand-name-google-chrome = Google Chrome
+banner-download-install-chrome-extension-headline = Try { -brand-name-relay } for { -brand-name-google-chrome }
+banner-download-install-chrome-extension-copy = The { -brand-name-firefox-relay } extension for { -brand-name-chrome } makes creating and using aliases even easier.
+banner-download-install-chrome-extension-cta = Get the { -brand-name-relay } extension
+multi-part-onboarding-premium-chrome-extension-get-title = Get the { -brand-name-relay } extension for { -brand-name-google-chrome }
+multi-part-onboarding-premium-chrome-extension-get-description = The { -brand-name-firefox-relay } extension for { -brand-name-chrome } makes creating and using email aliases even easier.
+multi-part-onboarding-premium-chrome-extension-button-download = Get { -brand-name-relay } Extension
+
```
[the remark this answers is not in the document]
Comment (Vinnl) [reply frontend/pendingTranslations.ftl:93]: This indeed does not go out to localizers; the existing process for that stays in place. It just makes it easier to review this when the l10n changes haven't been merged yet or content is even not final yet. Before this was in place, we'd either use a non-Fluent string, or put up with `???`s (which sometimes even accidentally shipped to prod) while waiting for that.

(These strings have, of course, by now been merged into the l10n repo, and will thus be overridden by the strings in there. Once we're at feature parity, I'll do a pass over this file and remove the strings that are no longer relevant, and submit the rest (mainly strings for screen readers in cases that the current website does not accommodate) to the l10n repo.)

File: frontend/src/components/dashboard/PremiumOnboarding.tsx (hunk 1 above)
Comment (Vinnl) [reply frontend/src/components/dashboard/PremiumOnboarding.tsx:392]: Love that you're jumping into the React code! Left a review there, overall approach looks good :)

File: frontend/src/components/dashboard/PremiumOnboarding.tsx [hunk 3]
```diff
@@ -384,21 +388,69 @@ const StepThree = () => {
...
+  if (isLargeScreen && supportsFirefoxExtension()) {
```
Comment (codemist) [inline frontend/src/components/dashboard/PremiumOnboarding.tsx:393]: Should the banners still be visible on mobile? I am still seeing the banners at a device width of < `500px`

Comment (codemist) [reply frontend/src/components/dashboard/PremiumOnboarding.tsx:393]: image

**Review by codemist — CHANGES_REQUESTED** [review]

File: frontend/src/components/dashboard/PremiumOnboarding.tsx (hunk 3 above)
Comment (Vinnl) [reply frontend/src/components/dashboard/PremiumOnboarding.tsx:393]: Oh, that's a good point! This file specifically is for the Premium onboarding flow, but the profile banners did indeed not yet look at screen size to determine which banners to show. Fixed in f6e61f6a.

**Review by groovecoder — APPROVED** [review]

**Pull request merged**